environments/logger/daemonset-logger.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: influxdb
+  namespace: fission
+spec:
+  ports:
+  - name: "8086"
+    port: 8086
+    targetPort: 8086
+    protocol: TCP
+
+---
+apiVersion: v1
```

Comment:
I'll duplicate the service files, one is for internal and the other is for external, cause user may want to use their own existing external db service.